sopel/plugins/rules.py

```diff
@@ -465,16 +465,25 @@ def __init__(self) -> None:
 
     def start(self) -> None:
         """Record a starting time (before execution)."""
-        self.started_at = datetime.datetime.utcnow()
+        self.started_at = datetime.datetime.utcnow().replace(tzinfo=datetime.timezone.utc)
 
     def end(self) -> None:
         """Record a ending time (after execution)."""
-        self.ended_at = datetime.datetime.utcnow()
+        self.ended_at = datetime.datetime.utcnow().replace(tzinfo=datetime.timezone.utc)
 
     def set_return_value(self, value: Any) -> None:
         """Set the last return value of a rule."""
         self.last_return_value = value
 
+    @property
+    def last_time(self):
+        # detect if we just started something or if it ended
```

Comment:
Public property should have a docstring.

```suggestion
        """Detect if we just started something or if it ended."""
```